pages/behaviors/manage.tsx

```diff
@@ -33,28 +32,53 @@ interface BehaviorAsProps {
     _id: string;
 }
 
-export default function manageBehaviorsPage({ behaviors }) {
+export default function manageBehaviorsPage() {
     // Verifies if user has the correct permissions
     const {allowed, role} = CheckUser(["Admin", "BCBA"])
     if(!allowed) return(<div>Redirecting...</div>);

```

Comment:
I don't think we need to fix this as part of the current story, but this line is causing a React error: 

Eventually we may just need to handle unauthorized redirects in the middleware of our next app, that way specific components won't have to be concerned with this. 

Warning: React has detected a change in the order of Hooks called by manageBehaviorsPage. This will lead to bugs and errors if not fixed. For more information, read the Rules of Hooks: https://reactjs.org/link/rules-of-hooks

   Previous render            Next render
   ------------------------------------------------------
1. useContext                 useContext
2. useContext                 useContext
3. undefined                  useState
   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^